src/lib/QtGui/QCursor/index.ts

```diff
@@ -0,0 +1,22 @@
+import addon from "../../core/addon";
+import { Component, NativeElement } from "../../core/Component";
+import { QPixmap } from "../QPixmap";
+
+type arg = NativeElement | number | QPixmap;
```

Comment:
I think this should be just number 
We could remove both NativeElement and QPixmap from here

Reply:
This actually was left intentionally because I intend to add full support for Pixmap/Masks cursors, just did not have time on hand right now.
Do you think this should be removed from this PR?

Reply:
No its okay. Let it be.

Reply:
I will merge this PR then ? @Uriziel01

Reply:
LGTM 👍